etc/vzlogger.conf.InfluxDB

```diff
@@ -0,0 +1,36 @@
+/**
+ * vzlogger configuration example for InfluxDB 
+ *
+ * use proper encoded JSON with javascript comments
+ *
+ * take a look at the wiki for detailed information:
+ * http://wiki.volkszaehler.org/software/controller/vzlogger#configuration
+*/
+
+{
+    // ... for general vzlogger settings see vzlogger.conf
+
+    "meters": [
+        // examples for InfluxDB as storage
+        {
+            // See vzlogger.conf for complete meter configuration options
+
+            "enabled": true,                 // disabled meters will be ignored
+            "protocol": "sml",               // see 'vzlogger -h' for list of available protocols
+            "device": "/dev/ttyAMA0",
+            "channels": [{
+                "api": "influxdb", // use the InfluxDB api
+                "uuid": "01234567-9abc-def0-1234-56789abcdefe", // use the uuid command to generate this
+                "identifier" : "1-0:16.7.0", // OBIS code for "power" 
+                "host": "127.0.0.1:8086",                       // This assumes that InfluxDB is running on localhost
```

Comment:
For consistency middleware would be better.

But i understand your opinion. Host would be better, even for volkszaehler and mysmartgrid.